daml-assistant/daml-helper/test-certificates/ca.crt

```diff
@@ -0,0 +1,29 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
I’ve followed the approach that we use for other TLS tests to copy the files in here for now. We might want to stop doing  that at some point but that seems like something for a separate PR.

Reply:
I wonder how we should change our approach to avoid this?

IMO we should have a plan / open an issue at least.

Reply:
My plan was to at least share the test certs across all tests as a next step. That makes them easy to update and avoids all the duplication. I’m not too worried about having them copied here provided we have instructions for how to generate them (e.g. the bash script in this PR) but we could factor it out into a `genrule` and avoid that. The main issue is (as usual) Windows, where we cannot simply get openssl from nix but need to figure out some other source. Haven’t looked into it, it might be very simple. But the ssl certs are valid for 10 years so I don’t think this is particularly urgent :slightly_smiling_face:

Reply:
Looks like scoop has openssl so hopefully things should be simple™ https://github.com/ScoopInstaller/Main/blob/master/bucket/openssl.json